web/src/components/TickerTable/index.tsx

```diff
@@ -41,21 +45,21 @@ export const TickerTable: React.FC<Props> = ({
                     </td>
                     <td>
                         <span>
-                            <Decimal fixed={market.amount_precision} thousSep=",">
+                            <Decimal fixed={!isFixedPrecision ? market.amount_precision : market.price_precision} thousSep=",">
```

Comment:
same for market.high, it's the highest price within 24h, so again for price we use price_precision.